### ISSUE TITLE
style img alt text

### ISSUE THREAD

**Issue by adammichaelwood** [opendatakit/website#98]
When an image is missing or broken, the `alt` text is displayed instead.
Currently, the alt text looks *exactly* the same as the surrounding body text, and (on some browsers) there is no indication that an image is missing. This leads to confusing, random sentences in the middle of text where an image was supposed to be.

See, for example:
http://opendatakit.org/2017/06/configure-collect-on-many-devices-with-qr-codes/

When viewed on Firefox:

![screen shot 2018-05-22 at 4 49 29 pm](https://user-images.githubusercontent.com/1776241/40396233-afa79a0e-5de0-11e8-9c4c-7ca5521b51b6.png)

**Comment by Ifyokoh** [comment]
Hi, is this issue still undone?, will like to work on it

**Comment by shobhitagarwal1612** [comment]
I have two ideas that can be applied to make the `alt` text different from normal text
 - adding a border to the image (gray colored)
 - changing the font color of alt text (red colored)
 -  both of the above

**Comment by shobhitagarwal1612** [comment]
![image](https://user-images.githubusercontent.com/8918023/47393380-e385b380-d73c-11e8-8a60-7b23f16dd44c.png)

@yanokwa Wha

**Comment by yanokwa** [comment]
https://mmistakes.github.io/minimal-mistakes/ is the theme we use. 

A look at https://mmistakes.github.io/minimal-mistakes/about/ suggests that there is a figcaption class. https://github.com/mmistakes/minimal-mistakes/search?q=figcaption&unscoped_q=figcaption shows the usage at https://mmistakes.github.io.  Can we just use that class and style?

**Comment by shobhitagarwal1612** [comment]
This is how the suggested class looks

**Comment by yanokwa** [comment]
In the spirit of https://github.com/opendatakit/website/pull/205, I'd make it sans-serif. Then maybe italicized and be done.

**Comment by shobhitagarwal1612** [comment]
@opendatakit-bot claim

**Comment by shobhitagarwal1612** [comment]
I guess bot isn't happy with me today

**Comment by shobhitagarwal1612** [comment]
![image](https://user-images.githubusercontent.com/8918023/47393928-9f93ae00-d73e-11e8-8d81-58d6a5819a09.png)

figcaption class along with sans-serif and italic font styles

**Comment by yanokwa** [comment]
That change looks good to me. It'd be good to fix that screenshot so we see it with an actual image though :)

**Comment by shobhitagarwal1612** [comment]
The screenshot is attached properly. I intentionally removed the image for testing the changes

**Comment by shobhitagarwal1612** [comment]
I have been using Firefox for testing this and didn't reproduce any missing image by default. I think the image attached with the issue was captured on a cloned repo without LFS files fetched, hence missing images

**Comment by shobhitagarwal1612** [comment]
Last idea before I send a PR:

![image](https://user-images.githubusercontent.com/8918023/47469566-291aad00-d81f-11e8-9957-9eff31ce11e9.png)

**CC** @lognaturel @yanokwa

**Comment by yanokwa** [comment]
Looks good to me. If/when you send in a PR, include a couple of screenshots of pages you've verified the change on.
